ethernet-view/src/components/ChartMenu/ChartList/ChartElement/ChartElement.tsx

```diff
@@ -0,0 +1,75 @@
+// @ts-ignore
+import CanvasJSReact from '@canvasjs/react-charts';
+import { ChartInfo, MeasurementInfo, Point } from "components/ChartMenu/types";
+import styles from "./ChartElement.module.scss";
+import { AiOutlineCloseCircle } from 'react-icons/ai'
+import { MutableRefObject, useCallback, useRef } from "react";
+
+var CanvasJSChart = CanvasJSReact.CanvasJSChart;
+
+const MAX_VALUE = 1000;
+
+type Props = {
+    chart: ChartInfo;
+    removeChart: (id: string) => void;
+    getMeasurementInfo: (id: string) => MeasurementInfo;
+    addMeasurementToChart: (chartId: string, measurementInfo: MeasurementInfo) => void;
+    removeMeasurementFromChart: (chartId: string, measurementName: string) => void;
```

Comment:
use the `ChartID` type alias instead of `string` and create type aliases for measurements IDs and Names